packages/react-core/src/components/JumpLinks/JumpLinks.tsx

```diff
@@ -10,6 +16,8 @@ export interface JumpLinksProps extends Omit<React.HTMLProps<HTMLElement>, 'labe
   isVertical?: boolean;
   /** Label to add to nav element. */
   label?: React.ReactNode;
+  /** Flag to always show the label when using `expandable` */
+  alwaysShowLabel?: boolean;
```

Comment:
Just my $0.02, I would expect that if a label is defined, it's shown, even when the jump links are expandable, and we would have a prop to only show the label as the toggle label in the expandable state (ie, hide it when the vertical tabs aren't expandable). At least that's how the core component is built - the class `.pf-m-toggle` is the opt-in to make the label only show as the toggle text when it's in the expandable layout. Wdyt @mcarrano @tlabaj?

Reply:
The consumer should be able to control this. Is that what the prop `alwaysShowLabel` does?

Reply:
Yep. I'm just asking what the default should be, and what the prop opts into now. Currently in this PR if you create jump links that are expandable on small screens, but regular vertical jump links on large screens, the jump links label you define will only show up in the expandable UI as the toggle text. It will not be visible on large screens.

I was suggesting that we leave the label as it is now, and if it's defined, it shows up as both the expandable toggle on small screens and the jump links label on large screens, and we create a prop to hide the label on large screens so that the label only serves as the expandable toggle text.

Reply:
I've set `alwaysShowLabel` to true by default.

In the future we could allow a different label for responsive or non-responsive views.